packages/manager/modules/anthos/src/dashboard/ips/template.html

```diff
@@ -0,0 +1,123 @@
+<div>
+
```

Comment:
Needed space ?

Reply:
For visibility, it's better to have a blank line.
Or may be you discuss on another aspect ?